swagger2.cabal

```diff
@@ -40,8 +40,8 @@ library
     Data.Swagger.Internal.Utils
     Data.Swagger.Internal.AesonUtils
   build-depends:       base        >=4.7   && <4.10
-                     , base-compat >=0.6.0 && <0.10
-                     , aeson
+                     , base-compat >=0.9.1 && <0.10
+                     , aeson       >=0.11.2.1
```

Comment:
needed for `ToJSON Day`, only `aeson-0.10` introduced it, but skipping directly to 0.11 should be a problem for any user.